src/reactivity/unwrap.ts

```diff
@@ -28,8 +29,12 @@ export function unwrapRefProxy(value: any, map = new WeakMap()) {
 
   for (const k of Object.keys(value)) {
     const r = value[k]
+    // don't unwrap or do anything to raw
+    if (isRaw(r)) {
```

Comment:
I found that when `r` is `null` or `undefined`, it causes `TypeError: Cannot convert undefined or null to object`.